### PR TITLE
Blog update

### PR DESCRIPTION
**Pull request by TeresaGobble**
## Description

This PR includes a sprint update for our site blog.

### CODE REVIEW

File: internal/faustjs.org/blog/2023-03-15-faust-update-mar-15-2023.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+slug: faust-update-mar-15-2023
+authors:
+  - name: Teresa Gobble
+    title: Associate Software Engineer
+    url: https://github.com/TeresaGobble
+    image_url: https://avatars.githubusercontent.com/u/50935135?v=4
+hide_table_of_contents: true
+---
+
+Hi folks, Terri here! The last two weeks have been all about improvements to the React-Gutenberg Bridge experience and to the Faust Admin Bar. Among other tasks to help polish these tools up, this sprint saw some great documentation additions, such as three different tutorials on how to create a block based off of core blocks, third party plugins, or custom blocks. The team also completed preparation for the DE{CODE} conference coming up next week- we hope to see you there!
```
Comment (josephfusco) [inline internal/faustjs.org/blog/2023-03-15-faust-update-mar-15-2023.md:13]:
```suggestion
Hi folks, Terri here! The last two weeks have been all about improvements to the React-Gutenberg Bridge experience and to the Faust Toolbar. Among other tasks to help polish these tools up, this sprint saw some great documentation additions, such as three different tutorials on how to create a block based off of core blocks, third party plugins, or custom blocks. The team also completed preparation for the DE{CODE} conference coming up next week- we hope to see you there!
```

File: internal/faustjs.org/blog/2023-03-15-faust-update-mar-15-2023.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+## What's next
+
+This next sprint we're looking ahead to what we want to see happen for the next quarter. Some of the goals the team is outlining include restructuring and improvements to Faust’s documentation to help streamline content, as well as reconfiguring the structure of the site itself to make navigation more intuitive and enjoyable. 
+
+We’ll be adding some polish to the [wp-graphql-content-blocks plugin](https://github.com/wpengine/wp-graphql-content-blocks) during this sprint too, including removing the plugin’s dependency on Composer’s autoloader, so the user can download the WordPress plugin from GitHub and install directly into WordPress without any developer actions. Control over the Faust Admin Bar’s visibility based on the WordPress user’s preference will also be a focus, as well as supporting visibility into Faust package's bundle sizes when various features are activated.
```
Comment (josephfusco) [inline internal/faustjs.org/blog/2023-03-15-faust-update-mar-15-2023.md:29]:
```suggestion
We’ll be adding some polish to the [wp-graphql-content-blocks plugin](https://github.com/wpengine/wp-graphql-content-blocks) during this sprint too, including removing the plugin’s dependency on Composer’s autoloader, so the user can download the WordPress plugin from GitHub and install directly into WordPress without any developer actions. Control over the toolbar’s visibility based on the WordPress user’s preference will also be a focus, as well as supporting visibility into Faust package's bundle sizes when various features are activated.
```

**Review by josephfusco — CHANGES_REQUESTED** [review]
Thanks for writing this one!

**Comment by TeresaGobble** [comment]
Thank you for fixing the toolbar name, @josephfusco! 😊

**Review by josephfusco — APPROVED** [review]

**Pull request merged**